protos/encodings.proto

```diff
@@ -151,6 +147,12 @@ message FixedSizeList {
   ArrayEncoding items = 2;
 }
 
+// An encoding for arrays that fit in a single buffer
+message Value {
+  // the buffer of values
+  BufferEncoding buffer = 1;
+}
```

Comment:
At first I was thinking this was like `Constant` encoding, but this is just a literal buffer? like a small bitmap?

Reply:
I almost want to call it `InlineBuffer` or something.

Reply:
Oh maybe I have this wrong. This is just a single leaf buffer, right? Maybe a better name is `SingleBuffer`?

Reply:
I guess there main facts here are:
* There is only one buffer
* Each value contains X bits
* Values are stacked one after the other in the buffer

E.g. roaring bitmap is just a single buffer but it is not "Value".  Also, you could compress a buffer and still have a single buffer but it is not "Value".  Maybe we need some collapsing here.

What if we get rid of `BufferEncoding`, `Flat`, `Bitmap` entirely and just replace `Value` with:

```
message Flat {
  uint64 bits_per_value = 1;
}
```

Then, on the code side of things, we could still have the same structure if we wanted, or we could collapse there as well.

Reply:
This seems clearer to me now.